app-backend/api/src/main/scala/config/Config.scala

```diff
@@ -1,30 +1,28 @@
 package com.azavea.rf.api.config
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.collection.JavaConversions._
 import com.azavea.rf.api.utils.Config
-import com.azavea.rf.api.AkkaSystem
-import com.azavea.rf.datamodel.FeatureFlag
 import com.azavea.rf.database.Database
 import com.azavea.rf.database.tables.FeatureFlags
-
+import com.azavea.rf.datamodel.FeatureFlag
 import io.circe.generic.JsonCodec
 
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Good call - I think the API server warrants a separate PR and issue https://github.com/azavea/raster-foundry/issues/2341